README.md

```diff
@@ -24,6 +24,23 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[VERSION]/wasi-sdk-[VERSION]-linux.tar.gz
+tar xvf wasi-sdk-[VERSION]-linux.tar.gz
+```
+
+## Use
+
+Use the clang installed in the wasi-sdk directory:
+```shell script
+CC="[WASI_SDK_PATH]/bin/clang --sysroot=[WASI_SDK_PATH]/share/wasi-sysroot"
+$CC foo.c -o foo.wasm
+```
+Note: `[WASK_SDK_PATH]/share/wasi-sysroot` contains the WASI-specific includes/libraries/etc.  
```

Comment:
The one thing I'd add here is that the clang comes preconfigured, I think to look for the sysroot under /opt/wasi-sdk, so if you install the packages there then you don't need to pass --sysroot.

Reply:
Ha! I think @sbc100 was commenting about this above... but in the opposite direction (i.e. more explicit use of `--sysroot`). I'll add a note here about `/opt/wasi-sdk`.